rllib/agents/ppo/ppo_torch_policy.py

```diff
@@ -42,7 +42,12 @@ def ppo_surrogate_loss(
         Union[TensorType, List[TensorType]]: A single loss tensor or a list
             of loss tensors.
     """
-    logits, state = model(train_batch)
+    if not isinstance(model, ModelV2):
+        logits, state, extra_outs = model(train_batch)
+        value_fn_out = extra_outs[SampleBatch.VF_PREDS]
+    else:
+        logits, state = model(train_batch)
+        value_fn_out = model.value_function()
```

Comment:
Another way of supporting this is to instead of adding if-else statements, wrap the raw torch model in a ModelV2 shim.

Reply:
That would avoid needing isinstance() everywhere.

Reply:
@sven1977 how about let's do what eric suggested?